json_serializable/lib/src/helper_core.dart

```diff
@@ -45,7 +45,7 @@ abstract class HelperCore {
   ///
   /// Returns the output of calling [genericClassArguments] with [element].
   @protected
-  String genericClassArgumentsImpl(bool withConstraints) =>
+  String genericClassArgumentsImpl({required bool withConstraints}) =>
```

Comment:
Ditto. I LOVE this refactor. Try to pull this out as tiny PRs. Just makes reviewing easier!